charts/consul/values.yaml

```diff
@@ -1847,6 +1847,12 @@ connectInject:
     # This value is also overridable via the "consul.hashicorp.com/transparent-proxy-overwrite-probes" annotation.
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
+  
+  disruptionBudget: 
```

Comment:
@jmurret Should the `enabled` be set to `true` by default or `false`?

Reply:
Please change it to true so it is the same as `server.disruptionBudget`.  The logic on that k8s resource is that it will only be installed if connectInject is true and this is true.

Reply:
Thank you!